schema-registry/rest-service/src/main/java/com/hortonworks/registries/schemaregistry/webservice/ConfluentSchemaRegistryCompatibleResource.java

```diff
@@ -212,9 +213,9 @@ public Response getSchemaVersion(@ApiParam(value = "subject", required = true)
                 }
                 SchemaVersionInfo fetchedSchemaVersionInfo = null;
                 try {
-                    Long id = Long.valueOf(versionId);
+                    Integer id = Integer.valueOf(versionId);
```

Comment:
maybe should be named version to avoid confusions with id which is global unique id.

Reply:
also would more match naming here for matching endpoint.

https://github.com/confluentinc/schema-registry/blob/8eb664dbc84b1c2db3666fa0771eeb0e0909f892/core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/SubjectVersionsResource.java

Reply:
@apsaltis Can you address this comment?

Reply:
@satishd Done -- sorry for missing that.